client/components/advanced-credentials/credentials-form/index.tsx

```diff
@@ -217,6 +217,7 @@ const ServerCredentialsForm: FunctionComponent< Props > = ( {
 					isError={
 						formErrors.kpri && ( interactions.kpri || ! formErrors.kpri.waitForInteraction )
 					}
+					data-private
```

Comment:
This is necessary to prevent LogRocket record this private key field

Reply:
![CleanShot 2024-04-07 at 20 57 57@2x](https://github.com/Automattic/wp-calypso/assets/1488641/9d7972f7-6ff6-42ff-87f2-1264fd2f2b2a)

Reply:
No longer needed, we are marking all input elements as private by default.